cluster-app-values-schema/README.md

```diff
@@ -0,0 +1,283 @@
+
+### R3: Array item schema must be defined {#r3}
+
+The items schema for all array properties MUST be defined using the `items` keyword.
+
+### R4: Properties must have a title {#r4}
+
+Each property MUST be annotated via the `title` keyword.
+
+The `title` keyword provides a user friendly label for the property, to be displayed in a user interface instead of the technical property name. The title should be chosen to match the wording used in documentation, especially in prosa text (as opposed to configuration examples).
+
+Example:
+
+```json
+{
+  ...
+  "properties": {
+    "name": {
+        "type": "string",
+        "title": "Cluster name",
+        ...
+    }
+  }
+}
+```
+
+Additional requirements apply to the title value:
+
+- The title MUST be written using sentence case capitalization. Correct: "Cluster name", incorrect: "Cluster Name".
+- The title MUST NOT contain punctuation marks, leading or trailing white space, control characters, tabs, nor multiple whitespaces in a row.
+- If the title annotates a property that is part of another object, the title SHOULD NOT include the parent property name, to avoid repetition.
+  - Example: with an object `/controlPlane` that is titled `Control plane`, the property  `/controlPlane/availabilityZones` should be titled `Availability zones`, not `Control plane availability zones`.
+
+### R5: Properties should have descriptions {#r5}
+
+Each property SHOULD be annotated and via the `description` keyword.
+
+Example:
+
+```json
+{
+  ...
+  "properties": {
+    "name": {
+        "type": "string",
+        "description": "Identifies this cluster uniquely within the installation.",
+        ...
+    }
+  }
+}
+```
+
+If a description is given, additional requirements apply to the value:
+
+- Description content MUST NOT include any line breaks.
+- Descriptions MUST NOT contain formatting code like e.g. Markdown or HTML, leading or trailing white space, control characters, tabs, nor multiple whitespaces in a row.
+- Descriptions MUST use sentence case capitalization and punctuation.
+- The description SHOULD NOT repeat the property name or title. Correct: "Identifies this cluster uniquely within the installation.", incorrect: "Cluster name identifies this cluster uniquely within the installation.".
+- Descriptions SHOULD be between 50 and 200 characters long.
+- Descriptions SHOULD be written in simple language.
+
+### R6: Properties should provide examples {#r6}
+
+Each property SHOULD provide one or more examples using the `examples` keyword.
+
+Example:
+
+```json
+{
+  ...
+  "properties": {
+    "name": {
+        "type": "string",
+        "examples": ["devel001"],
+        ...
+    }
+  }
+}
+```
+
+Best practices:
+
+- Ideally, provide one example.
+- Provide more than one example if you want to highlight that very different values are accepted.
+- Provide no more than five examples
```

Comment:
Should this be a MUST/SHOULD rather than a best practice?

Reply:
Will clarify this.